src/test/java/pages/LoginPage.java

```diff
@@ -1,52 +1,35 @@
 package pages;
-
-import org.apache.commons.logging.Log;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-
-public class LoginPage extends BasePage{
-
-    //locators
-    By submitButtonLocator = By.cssSelector("[type='submit']");
-    By emailField = By.cssSelector("[type='email']");
-    By passwordField = By.cssSelector("[type='password']");
-
-    By profileIcon  = By.xpath("//a[@class='view-profile']");
-
 
+public class LoginPage extends BasePage {
     public LoginPage( WebDriver givenDriver) {
-        super(givenDriver);
-    }
-
-    public LoginPage clickSubmitBtn() {
-        driver.findElement(submitButtonLocator).click();
-        return this;
-    }
+        super(givenDriver);}
+    @FindBy (css="[type='email']")
+    WebElement emailFieldLocator;
 
-    public LoginPage provideEmail(String email) {
-        WebElement emailElement = driver.findElement(emailField);
-        emailElement.sendKeys(email);
-        return this;
-    }
+    @FindBy(css="[type='password']")
+    WebElement passwordFieldLocator;
 
-    public LoginPage providePassword(String password) {
-        WebElement passwordElement = driver.findElement(passwordField);
-        passwordElement.sendKeys(password);
+    @FindBy(css ="button[type='submit']")
+   WebElement submitButtonLocator;
+    public LoginPage provideEmail(String email){
+        emailFieldLocator.sendKeys(email);
         return this;
     }
-
-    public LoginPage logIn(){
-        provideEmail("demo@class.com");
-        providePassword("te$t$tudent");
-        clickSubmitBtn();
-        return this;
+public LoginPage providePassword (String password) {
+    passwordFieldLocator.sendKeys(password);
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)